### PR TITLE
Changing userId to be a string instead of an integer

### PR DESCRIPTION
**Pull request by edblankenship**
UserId’s can be non-numeric with Intercom.  Updating the GET method to
allow for string values.  Hope this helps!

### CODE REVIEW

File: IntercomDotnet/Resources/Users.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ public dynamic Get(string email = null, int? userId = null)
                         request.AddParameter("email", email);
 
                     if (userId != null)
```
Comment (mgoodfellow) [inline IntercomDotnet/Resources/Users.cs]: This should probably become:

``` csharp
if (!string.IsNullOrEmpty(userId))
```

?

File: IntercomDotnet/Resources/Users.cs [hunk 2]
```diff
@@ -26,7 +26,7 @@ public dynamic Delete(object hash)
                 });
         }
 
-        public dynamic Get(string email = null, int? userId = null)
+        public dynamic Get(string email = null, string userId = null)
```
Comment (mgoodfellow) [inline IntercomDotnet/Resources/Users.cs:36]: Is it worth providing this as an overload? That would stop any breakages for existing users if they update the library. The old function (this one) can be marked as `[Obsolete]` to get users to move over in time?

**Comment by mgoodfellow** [comment]
Hi @EdBlankenship,

Thanks for the PR! Can you have a look at my comments? I'm away this weekend but can merge on Monday,

Thanks!

File: IntercomDotnet/Resources/Users.cs (hunk 2 above)
Comment (edblankenship) [inline IntercomDotnet/Resources/Users.cs:36]: Great suggestion!  Thanks for the catch.

**Comment by edblankenship** [comment]
@mgoodfellow - No problem!  Great suggestions.  New updates coming shortly.

**Comment by mgoodfellow** [comment]
Hi @EdBlankenship,

Thanks for the changes, LGTM :+1: 

Resolves: https://github.com/tarr11/IntercomDotnet/issues/15

I will merge this now and update the package on NuGet

**Pull request merged**